infrastructures/docker/infrastructure/src/main/java/org/eclipse/che/workspace/infrastructure/docker/provisioner/labels/SinglePortLabelsProvisioner.java

```diff
@@ -94,10 +95,11 @@ public void provision(DockerEnvironment internalEnv, RuntimeIdentity identity)
    * exec-agent-http-dev-machine-workspaceao6k83hkdav975g5
```

Comment:
Now it will be `exec-agent-http_dev-machine_workspaceao6k83hkdav975g5`

Reply:
fixed